README.md

```diff
@@ -95,6 +130,7 @@ or get started by trying out the example:
 
 ``` r
 library(mmrm)
+#> Warning: package 'mmrm' was built under R version 4.2.3
```

Comment:
I guess we should suppress warnings in the `Rmd` file chunk options here to avoid this warning